### PR TITLE
Add new WebContainer message

### PR DESCRIPTION
**Pull request by pmd1nh**

### CODE REVIEW

**Comment by pmd1nh** [comment]
@dmuelle These messages are from https://github.com/OpenLiberty/open-liberty/pull/24722

I need to separate them into this new PR so I can have it merge by Mar 28 in order for the translation to complete for 24.0.0.4.  I have consulted with Sam Wong.  The main fix is not ready by 28th.  Thanks!

**Comment by pmd1nh** [comment]
#build

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -188,3 +188,6 @@ excludeAllHandledTypesClasses.desc=When this property is set to true, during sta
 
 skipEncodedCharVerification.name=Skip verification of encoded characters in URI
 skipEncodedCharVerification.desc=A request is rejected if the request URI contains the %23, %2E, %2F, or %5C encoded characters. When this property is set to true, the request URI is not checked for these encoded characters. This property is available starting in servlet 6.0, and default value is false.
+
+maxFileCount.name=The maximum uploaded files in a single request
+maxFileCount.desc=The maximum number of files that can be uploaded by a multipart/form-data request. The default is 5000; setting to -1 for unlimited files upload.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: ID suggested rewrite:
maxFileCount.desc=The maximum number of files that can be uploaded by a multipart/form-data request. The default value is 5000. For unlimited file upload, set the value to -1.

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 2]
```diff
@@ -1622,3 +1622,7 @@ uri.has.fragment.character=URI has a fragment [#] character; encoded [%23] or no
 uri.has.dot.character=URI has encoded dot [%2E] character.
 uri.has.forwarslash.character=URI has encoded forward slash [%2F] character.
 uri.has.backslash.character=URI has backslash character; encoded [%5C] or not.
+
+multipart.file.count.max.exceeded=CWWWC0006E: The number of uploaded files in the request exceeds the maximum permitted number, which is [{0}]. 
+multipart.file.count.max.exceeded.explanation=The number of files that are uploaded by the multipart/form-data request exceeded the maximum limit.
+multipart.file.count.max.exceeded.useraction=Increase the maximum file count by setting the maxFileCount webContainer custom property in the server.xml file. The default maximum is 5000; setting to -1 for unlimited files upload.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: Suggested ID rewrite:
multipart.file.count.max.exceeded.useraction=Increase the maximum number of files that can be uploaded per request by setting the maxFileCount webContainer custom property in the server.xml file. The default value is 5000. For unlimited file upload, set the value to -1.

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 3]
```diff
@@ -1622,3 +1622,7 @@ uri.has.fragment.character=URI has a fragment [#] character; encoded [%23] or no
 uri.has.dot.character=URI has encoded dot [%2E] character.
 uri.has.forwarslash.character=URI has encoded forward slash [%2F] character.
 uri.has.backslash.character=URI has backslash character; encoded [%5C] or not.
+
+multipart.file.count.max.exceeded=CWWWC0006E: The number of uploaded files in the request exceeds the maximum permitted number, which is [{0}]. 
+multipart.file.count.max.exceeded.explanation=The number of files that are uploaded by the multipart/form-data request exceeded the maximum limit.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: suggested ID rewrite:
multipart.file.count.max.exceeded.explanation=The number of files to be uploaded by the multipart/form-data request exceeded the set limit.

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 4]
```diff
@@ -1622,3 +1622,7 @@ uri.has.fragment.character=URI has a fragment [#] character; encoded [%23] or no
 uri.has.dot.character=URI has encoded dot [%2E] character.
 uri.has.forwarslash.character=URI has encoded forward slash [%2F] character.
 uri.has.backslash.character=URI has backslash character; encoded [%5C] or not.
+
+multipart.file.count.max.exceeded=CWWWC0006E: The number of uploaded files in the request exceeds the maximum permitted number, which is [{0}].
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: ID suggested rewrite:
multipart.file.count.max.exceeded=CWWWC0006E: The request contained more files for upload than is permitted. The maximum number of files is [{0}].

**Review by jantley-ibm — COMMENTED** [review]
I've suggested some rewording; let me know if you have any questions or concerns about my proposed text. When we agree on changes I'll add ID reviewed label. Thanks!

**Comment by pmd1nh** [comment]
@jantley-ibm I updated the messages.  Thanks!

**Comment by pmd1nh** [comment]
#build

**Comment by pmd1nh** [comment]
@jantley-ibm Can you please approve this PR so I can merge for translation Dcut tomorrow.  Thanks!

**Comment by pmd1nh** [comment]
#libby

**Comment by posmith** [comment]
L2 message review completed

**Review by epj — APPROVED** [review]
approved for translation

**Pull request merged**